src/main/java/Attendance.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+
+public class Attendance {
+		ArrayList<Student> students = new ArrayList<Student>();
+		Course course;
+		Teacher teacher;
+		boolean present;
+
+		public Attendance(ArrayList<Student> student, Course course, Teacher teacher, boolean present) {
+			this.students = student;
+			this.course = course;
+			this.teacher = teacher;
+			this.present = present;
```

Comment:
Do we need this present ? 
We also have to make sure to capture the date, In the event or maybe somewhere else.